scanner/e2etests/testdata/image_tests.json

```diff
@@ -1532,8 +1532,7 @@
                 "NamespaceName": "ubuntu:22.10",
                 "Version": "3.0.5-2ubuntu2",
                 "AddedBy": "sha256:2b441754735ea7decb684ef19d54115fc309c270fe7b87ed36aa3773ce50b78b",
-                "Location": "var/lib/dpkg/status",
-                "FixedBy": "3.0.5-2ubuntu2.3"
```

Comment:
So, no fixable vulns on this image anymore?

Reply:
right, this is ubuntu 22.10 which is no longer supported